pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarSplitManager.java

```diff
@@ -328,6 +328,9 @@ Collection<PulsarSplit> getSplitsForTopic(String topicNamePersistenceEncoding,
                 splits.add(pulsarSplit);
             }
             return splits;
+        } catch (ManagedLedgerException exception){
+            log.error(exception, "Open cursor failed");
```

Comment:
Do you want to catch the case that the managed ledger of the topic is not existing yet? Many exceptions extend `ManagedLedgerException`, maybe we don't need to catch all `ManagedLedgerException`.